judge/testcase_run.sh

```diff
@@ -212,6 +212,12 @@ runcheck "$RUN_SCRIPT" $RUNARGS \
 	--stderr=program.err --outmeta=program.meta -- \
 	"$PREFIX/$PROGRAM" 2>runguard.err
 
+if [ "$CREATE_WRITABLE_TEMP_DIR" ]; then
+	# Revoke access to the TMPDIR as security measure
+	chown -R root:root "$TMPDIR"
```

Comment:
```suggestion
	chown -R "$(id -un):" "$TMPDIR"
```
This is what I would do, but I would like to get Jaap's opinion as well.

Reply:
Lets see what @eldering thinks.

Reply:
Looks ok to me.